pygerm/tests/test_hdf5germ.py

```diff
@@ -0,0 +1,28 @@
+# some setup
```

Comment:
oops need to remove this. will later. it's unclear to me if we want to turn this into a test for the older hdf5 format

Reply:
The long-term format is not clear.  

Currently this is a R&D project and the binary format is what the detector group is used to working with.  Long term (when this passes to being a production detector at XPD) we may want to go back to the hdf5 format (at it plays much nicer with the data broker and gives fast column-wise access).